src/App/Config/constants.ts

```diff
@@ -6,6 +6,11 @@ const BaseConfig = {
 	APP_NAME: '<APP_NAME>',
 	APP_SHORT_NAME: '<APP_SHORT_NAME>',
 
+	Github: {
+		api: 'https://api.github.com',
+		authorization: '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',
```

Comment:
why did you use the authorization token?

Reply:
you can easily request a public repository without any token, like this:
`https://api.github.com/repos/openscilab/pymilo`
putting an access token in a public frontend repository isn't safe at all, since it is getting exposed to everybody, even right now!
you have to first of all invalidate the token and surely remove the associated commit(commits containing the token) and possibly you have to force rebase.
@F-Shahali

Reply:
[According to details included, the communication thread continued in Telegram Private Chat]